pkg/data/account.go

```diff
@@ -0,0 +1,146 @@
+package data
+
+import (
+	gErrors "errors"
+	"fmt"
+	"strconv"
+
+	"github.com/Optum/dce/pkg/errors"
+	"github.com/Optum/dce/pkg/model"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+
+// Account - Data Layer Struct
+type Account struct {
+	DynamoDB       dynamodbiface.DynamoDBAPI
+	TableName      string `env:"ACCOUNT_DB"`
+	ConsistentRead bool   `env:"USE_CONSISTENT_READS" envDefault:"false"`
+}
+
+// WriteAccount the Account record in DynamoDB
```

Comment:
This could probably use some expanded comments, 
since some of the behavior might not be assumed, and it's core to the system.
eg., that:
- It's an "upsert" operation (creates or updates existing)
- we assume it's a create if `lastModified` is nil